xml/System.Linq/ParallelEnumerable.xml

```diff
@@ -1220,7 +1220,7 @@
         <returns>The average of the sequence of values.</returns>
         <remarks>To be added.</remarks>
         <exception cref="T:System.OperationCanceledException">The query was canceled with the token passed in through <paramref name="WithCancellation" />.</exception>
-        <exception cref="T:System.ArgumentNullException">A sequence of values that are used to calculate an average.The average of the sequence of values.<paramref name="source" /> is a null reference (Nothing in Visual Basic).</exception>
+        <exception cref="T:System.ArgumentNullException">A sequence of values that are used to calculate an average. The average of the sequence of values.<paramref name="source" /> is a null reference (Nothing in Visual Basic).</exception>
```

Comment:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="source" /> is a null reference (Nothing in Visual Basic).</exception>
```

or

```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="source" /> is a <see langword="null"/>.</exception>
```

Reply:
Good catch. I didn't know you could have more than one suggestion like that. Cool!